doc/website/_docs/A01-sample-example.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "ROHD Example"
+permalink: /docs/sample-example/
+excerpt: "Sample example of using ROHD."
+last_modified_at: 2022-12-05T08:48:05-04:00
+toc: true
+---
+
+Please make sure you had setup your ROHD Framework before running the test. Information of installation can be found at [setup page](/get-started/setup/).
+
+## A full example of a counter module
+
+To get a quick feel for what ROHD looks like, below is an example of what a simple counter module looks like in ROHD.
+
+```dart
+// Import the ROHD package
```

Comment:
Its kinda annoying how narrow the content is sometimes when the code doesn't fit.  Should we reformat the code to be narrower or make the font smaller?  The Dart autoformatter makes it less than 80 chars wide, but does the default width even fit that?

Reply:
I add `.sass` configuration to reduce the font size. Look better now and the code can be fit into a snippet.